common/changes/@uifabric/set-version/version_2018-09-27-16-50.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/set-version",
+      "comment": "version files now expose a call that can be called by apps that don't do index import",
+      "type": "patch"
+    }
+  ],
+  "packageName": "@uifabric/set-version",
+  "email": "kchau@microsoft.com"
+}
```

Comment:
Should we update the baseline so it more accurately reflects this not-insignificant change?

Reply:
Okay I've reverted that generator change so this isn't necessary anymore